godel_surface_detection/launch/kinect.launch

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
```

Comment:
Will this actually work? The static tf publisher makes me suspicious. This goes again to the point of documenting what our TF interfaces are, if you will.